Sources/TreeSitterClient/TreeSitterClient.swift

```diff
@@ -117,11 +117,7 @@ extension TreeSitterClient {
                                  limit: Int,
                                  readHandler: @escaping Parser.ReadBlock,
                                  completionHandler: @escaping () -> Void) {
-        guard let oldEndPoint = oldEndPoint else {
-            assertionFailure("oldEndPoint unavailable")
-            return
-        }
```

Comment:
This assertion should probably remain if locationTransformer is non-nil.

Reply:
Updated code for this change